packages/components/.mocharc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "require": ["ignore-styles"],
+  "require": ["ignore-styles", "source-map-support/register"],
```

Comment:
Is `source-map-support/register` is not needed when we call `sourceMapSuppor.install()` in setup file.

Reply:
I'm going to remove this altogether when https://github.com/iTwin/presentation/pull/528 merges.